recipes/spirv-tools/all/conanfile.py

```diff
@@ -74,16 +74,21 @@ def export(self):
         copy(self, f"dependencies/{self._dependencies_filename}", self.recipe_folder, self.export_folder)
 
     def export_sources(self):
-        for p in self.conan_data.get("patches", {}).get(self.version, []):
-            copy(self, p["patch_file"], self.recipe_folder, self.export_sources_folder)
+        export_conandata_patches(self)
 
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
+            try:
+                del self.options.fPIC
+            except Exception:
+                pass
```

Comment:
Now available!

```suggestion
            del self.options.rm_safe("fPIC")
```

Reply:
Are you sure? https://github.com/conan-io/conan-center-index/pull/13851#issuecomment-1295802373